Protoscope/ConsoleView.swift

```diff
@@ -40,7 +40,7 @@ class ConsoleView: UIView {
 		textView.textStorage.deleteCharactersInRange(NSMakeRange(0, textView.textStorage.length))
 	}
 
-	override init() {
```

Comment:
My guess is because `init` is not a designated initializer of `UIView` but it's just a guess, and they didn't document it.